system/BaseSpec.cfc

```diff
@@ -32,15 +32,9 @@ component {
 	// Focused Structures
 	this.$focusedTargets       = { "suites" : [], "specs" : [] };
 
-	// Setup Request Utilities
+	// Setup Request Utilities struct
 	if ( !request.keyExists( "testbox" ) ) {
-		request.testbox = {
-			"console"          : variables.console,
-			"debug"            : variables.debug,
-			"clearDebugBuffer" : variables.clearDebugBuffer,
-			"print"            : variables.print,
-			"println"          : variables.println
-		};
+		request.testbox = {};
```

Comment:
Param the struct with the very first spec to be run, just incase any users of TestBox have written code that depends on the existence of the struct.

Should I add no-op functions or something in here to preserve backwards compatibility?  These functions will exist as soon as the first spec begins to run.